Sources/HipMobileUI/ViewModels/Pages/AppetizerPageViewModel.cs

```diff
@@ -91,6 +100,46 @@ public AppetizerPageViewModel(Exhibit exhibit)
             await Navigation.PushAsync(downloadPage);
         }
 
+        /// <summary>
+        /// Sets the average user rating, the count and the stars if the user is connected to the internet.
+        /// </summary>
+        /// <returns></returns>
+        private async void SetUserRating()
```

Comment:
Maybe call this `RefreshRating` to make more clear that this method does something on the network? Just an idea.

Reply:
Changed the name to RefreshUserRating